clients/src/main/java/org/oracle/okafka/clients/admin/DescribeTopicsResult.java

```diff
@@ -28,11 +28,11 @@
 
 import org.apache.kafka.clients.admin.TopicDescription;
 import org.apache.kafka.common.KafkaFuture;
+import org.apache.kafka.common.Uuid;
 
 public class DescribeTopicsResult extends org.apache.kafka.clients.admin.DescribeTopicsResult {
 
-	protected DescribeTopicsResult(Map<String, KafkaFuture<TopicDescription>> futures) {
```

Comment:
Do we not need both the constructors ?

Reply:
yes i will add separate constructor for idFuture and nameFuture, that will be more systematic. currently i am passing null to one of the values.